source/pom.xml

```diff
@@ -35,15 +35,12 @@
   <description>The file server Parent</description>
 
   <properties>
-    <java.version>17</java.version>
+    <java.version>21</java.version>
```

Comment:
Java version can probably be left out? as it will be inherited from the `root-pom`?